homeassistant/components/overkiz/cover_entities/vertical_cover.py

```diff
@@ -99,3 +100,37 @@ async def async_close_cover(self, **kwargs: Any) -> None:
         """Close the cover."""
         if command := self.executor.select_command(*COMMANDS_CLOSE):
             await self.executor.async_execute_command(command)
+
+
+class LowSpeedCover(VerticalCover):
+    """Representation of an Overkiz Low Speed cover."""
+
+    def __init__(
+        self,
+        device_url: str,
+        coordinator: OverkizDataUpdateCoordinator,
+    ) -> None:
+        """Initialize the device."""
+        super().__init__(device_url, coordinator)
+        self._attr_name = f"{self._attr_name} Low Speed"
+        self._attr_unique_id = f"{self._attr_unique_id}_low_speed"
+
+    async def async_set_cover_position(self, **kwargs: Any) -> None:
+        """Move the cover to a specific position."""
+        await self.async_set_cover_position_low_speed(**kwargs)
+
+    async def async_open_cover(self, **kwargs: Any) -> None:
+        """Open the cover."""
+        await self.async_set_cover_position_low_speed(**{ATTR_POSITION: 100})
+
+    async def async_close_cover(self, **kwargs: Any) -> None:
+        """Close the cover."""
+        await self.async_set_cover_position_low_speed(**{ATTR_POSITION: 0})
+
+    async def async_set_cover_position_low_speed(self, **kwargs: Any) -> None:
+        """Move the cover to a specific position with a low speed."""
+        position = 100 - kwargs.get(ATTR_POSITION, 0)
+
+        await self.executor.async_execute_command(
+            OverkizCommand.SET_CLOSURE_AND_LINEAR_SPEED, position, "lowspeed"
```

Comment:
```suggestion
            OverkizCommand.SET_CLOSURE_AND_LINEAR_SPEED, position, OverkizCommandParam.LOW_SPEED
```

Would be good to add this command param to pyoverkiz enums and type it here.

Reply:
You mean this? https://github.com/iMicknl/python-overkiz-api/pull/383/files :)